### PR TITLE
Dts 29305 sonarqube issue fix

### PR DESCRIPTION
**Pull request by purushottam08**
sonarqube issue fixed in jira processor

### CODE REVIEW

File: processors/azure-boards/src/main/java/com/publicissapient/kpidashboard/azure/client/azureissue/KanbanAzureIssueClientImpl.java [hunk 1]
```diff
@@ -535,16 +531,11 @@ private void addHistoryInAzureIssue(KanbanIssueCustomHistory azureIssueCustomHis
...
 	 * @param updateValueList
 	 *            Change Log list
-	 * @param fieldMapping
-	 *            List of JiraIssueCustomHistory
 	 */
-	private void createKanbanIssueHistory(KanbanIssueCustomHistory azureIssueCustomHistory, KanbanJiraIssue azureIssue,
-			List<com.publicissapient.kpidashboard.common.model.azureboards.updates.Value> updateValueList,
-			FieldMapping fieldMapping) {
+	private void createKanbanIssueHistory(KanbanIssueCustomHistory azureIssueCustomHistory,
+			List<com.publicissapient.kpidashboard.common.model.azureboards.updates.Value> updateValueList) {
```
Comment (aksshriv1) [inline processors/azure-boards/src/main/java/com/publicissapient/kpidashboard/azure/client/azureissue/KanbanAzureIssueClientImpl.java:538]: check import optimization and formating

File: processors/azure-boards/src/main/java/com/publicissapient/kpidashboard/azure/client/azureissue/KanbanAzureIssueClientImpl.java [hunk 2]
```diff
@@ -668,9 +658,8 @@ private void saveKanbanAccountHierarchy(List<KanbanJiraIssue> jiraIssueList, Pro
 				List<KanbanAccountHierarchy> additionalFiltersHierarchies = accountHierarchiesForAdditionalFilters(
 						kanbanJiraIssue, projectHierarchy, projectHierarchyLevel, hierarchyLevelList);
 
-				additionalFiltersHierarchies.forEach(accountHierarchy -> {
-					accHierarchyToSave(accountHierarchy, existingKanbanHierarchy, accHierarchyToSave);
-				});
+				additionalFiltersHierarchies.forEach(accountHierarchy -> accHierarchyToSave(accountHierarchy,
```
Comment (aksshriv1) [inline processors/azure-boards/src/main/java/com/publicissapient/kpidashboard/azure/client/azureissue/KanbanAzureIssueClientImpl.java:661]: check formating

**Review by hirbabar — DISMISSED** [review]

**Review by eswbogol — APPROVED** [review]

File: processors/azure-boards/src/main/java/com/publicissapient/kpidashboard/azure/client/azureissue/KanbanAzureIssueClientImpl.java (hunk 1 above)
Comment (purushottam08) [reply processors/azure-boards/src/main/java/com/publicissapient/kpidashboard/azure/client/azureissue/KanbanAzureIssueClientImpl.java:538]: resolved

File: processors/azure-boards/src/main/java/com/publicissapient/kpidashboard/azure/client/azureissue/KanbanAzureIssueClientImpl.java (hunk 2 above)
Comment (purushottam08) [reply processors/azure-boards/src/main/java/com/publicissapient/kpidashboard/azure/client/azureissue/KanbanAzureIssueClientImpl.java:661]: resolved

**Review by aksshriv1 — DISMISSED** [review]

**Review by hargupta15 — APPROVED** [review]

**Pull request merged**